base/timing.jl

```diff
@@ -169,13 +175,16 @@ julia> @time begin
 macro time(ex)
     quote
         while false; end # compiler heuristic: compile this block (alter this if the heuristic changes)
+        reset_cumulative_compile_time() # reduce risk of overflow on cumulative counter
```

Comment:
This breaks nesting.

Reply:
Ah yes. Perhaps it could go at parse-time

Reply:
It'll just cause more strange corner cases. There's no need for this. It won't overflow before time_ns does.

Reply:
Good point. Removed 👍🏻